### PR TITLE
✨ RTC vendors: add query Param for Consent and ConsetMetadata fields to PubMatic OpenWrap

### PR DESCRIPTION
**Pull request by pm-shashank-jain**
* Added query param for CONSENT_STRING
* Added query params using CONSENT_METADATA for following fields
  * consentTypeString
  * gdprApplies
  * addtl_consent

REF:https://github.com/ampproject/amphtml/issues/22917, https://github.com/ampproject/amphtml/issues/30006

### CODE REVIEW

**Review by lannka — APPROVED** [review]

File: src/service/real-time-config/callout-vendors.js [hunk 1]
```diff
@@ -155,10 +155,10 @@ const RTC_VENDORS = jsonConfiguration({
     disableKeyAppend: true,
   },
   openwrap: {
-    // PubMatic OpenWrap
+    // PubMatic OpenWrap
```
Comment (lannka) [inline src/service/real-time-config/callout-vendors.js:158]: remove trailing space

File: src/service/real-time-config/callout-vendors.js [hunk 2]
```diff
@@ -155,10 +155,10 @@ const RTC_VENDORS = jsonConfiguration({
     disableKeyAppend: true,
   },
   openwrap: {
-    // PubMatic OpenWrap
+    // PubMatic OpenWrap 
     url:
-      'https://ow.pubmatic.com/amp?v=1&w=ATTR(width)&h=ATTR(height)&ms=ATTR(data-multi-size)&auId=ATTR(data-slot)&purl=HREF&pubId=PUB_ID&profId=PROFILE_ID',
-    macros: ['PUB_ID', 'PROFILE_ID'],
+      'https://ow.pubmatic.com/amp?v=1&w=ATTR(width)&h=ATTR(height)&ms=ATTR(data-multi-size)&auId=ATTR(data-slot)&purl=HREF&pubId=PUB_ID&profId=PROFILE_ID&consent_string=CONSENT_STRING&gdpr_applies=CONSENT_METADATA(gdprApplies)&addtl_consent=CONSENT_METADATA(additionalConsent)&consent_type=CONSENT_METADATA(consentStringType)',
+    macros: ['PUB_ID', 'PROFILE_ID','CONSENT_STRING'],
```
Comment (lannka) [inline src/service/real-time-config/callout-vendors.js:161]: add space after `,` should fix your lint error

File: src/service/real-time-config/callout-vendors.js (hunk 1 above)
Comment (pm-shashank-jain) [reply src/service/real-time-config/callout-vendors.js]: Removed trailing space!

File: src/service/real-time-config/callout-vendors.js (hunk 2 above)
Comment (pm-shashank-jain) [reply src/service/real-time-config/callout-vendors.js]: Added an space.

Comment (pm-shashank-jain) [reply src/service/real-time-config/callout-vendors.js]: Thanks

**Review by lannka — APPROVED** [review]

**Comment by pm-harshad-mane** [comment]
Hello @lannka 

PubMatic had signed the CLA in past for my user-id (pm-harshad-mane), can we use the same?

**Comment by pm-shashank-jain** [comment]
@lannka Can you please merge the required changes

**Comment by lannka** [comment]
sorry for the delay. could you resolve the conflict and try again?

**Pull request closed without merging**